### ISSUE TITLE
GCparagon installation fails through mamba/conda

### ISSUE THREAD

**Issue by Rociocc93** [BGSpiegl/GCparagon#3]
Hello!

I tried to install GCparagon into my system (Ubuntu 20.04) from a freshly cloned repository using mamba as recommended, but the environment cannot be solved because of package incompatibility. I have also tried the installation using conda with the classic solver and with the libmamba one, resulting in the same issue.
Any ideas on why this might be happening?

Thanks in advance,

Rocío

**Comment by BGSpiegl** [comment]
Hi Rocio,
I developed GCparagon on Ubuntu 22.04.3 LTS (64 bit), mamba 1.4.2, Conda 23.3.1.
Which versions do you have?
`mamba --version`
I do not have a Ubuntu 20 system available at the moment to try and reproduce the error
Could you provide the mamba/conda error output?
Another way to potentially resolve the issue could be upgrading your Ubuntu installation to 22.

**Comment by Rociocc93** [comment]
Hi,

Thanks for your fast reply!

The versions are:
- mamba 1.5.4
- conda 23.7.4

The conda error output (libmamba solver): 

LibMambaUnsatisfiableError: Encountered problems while solving:
  - package python-kaleido-0.2.1-py39h06a4308_0 is excluded by strict repo priority
  - package pysam-0.19.1-py27h7835474_1 requires libdeflate >=1.13,<1.14.0a0, but none of the providers can be installed
  - package requests-2.28.0-pyhd8ed1ab_0 requires urllib3 >=1.21.1,<1.27, but none of the providers can be installed
  - package samtools-1.16.1-h6899075_0 requires htslib >=1.16,<1.19.0a0, but none of the providers can be installed
  - nothing provides openssl >=1.1.1,<1.1.2.0a0 needed by python-2.7.15-h9bab390_2

Could not solve for environment specs
The following packages are incompatible
├─ pybedtools is installable with the potential options
│  ├─ pybedtools [0.6.9|0.7.10|...|0.8.1] would require
│  │  └─ python [2.7* |>=2.7,<2.8.0a0 ] with the potential options
│  │     ├─ python [2.7.12|2.7.13|...|2.7.18], which can be installed;
│  │     └─ python [2.7.15|3.6.7|3.7.1] would require
│  │        └─ openssl >=1.1.1,<1.1.2.0a0 , which does not exist (perhaps a missing channel);
│  ├─ pybedtools [0.6.9|0.8.1|0.8.2] would require
│  │  ├─ python >=2.7,<2.8.0a0  with the potential options
│  │  │  ├─ python [2.7.12|2.7.13|...|2.7.18], which can be installed;
│  │  │  └─ python [2.7.15|3.6.7|3.7.1], which cannot be installed (as previously explained);
│  │  └─ python_abi 2.7.* *_cp27mu, which can be installed;
│  ├─ pybedtools [0.7.0|0.7.2|...|0.7.9] would require
│  │  └─ python 3.4* , which can be installed;
│  ├─ pybedtools [0.7.10|0.7.4|...|0.8.0] would require
│  │  └─ python [3.5* |>=3.5,<3.6.0a0 ], which can be installed;
│  ├─ pybedtools [0.7.10|0.7.9] would require
│  │  └─ python 3.6*  with the potential options
│  │     ├─ python [2.7.15|3.6.7|3.7.1], which cannot be installed (as previously explained);
│  │     ├─ python [3.6.0|3.6.1|...|3.6.9], which can be installed;
│  │     └─ python [3.6.0a3|3.6.0a4|...|3.6.0rc1], which can be installed;
│  ├─ pybedtools [0.7.10|0.8.0|0.8.1] would require
│  │  └─ python >=3.6,<3.7.0a0  with the potential options
│  │     ├─ python [2.7.15|3.6.7|3.7.1], which cannot be installed (as previously explained);
│  │     └─ python [3.6.0|3.6.1|...|3.6.9], which can be installed;
│  ├─ pybedtools [0.8.0|0.8.1] would require
│  │  └─ python >=3.7,<3.8.0a0  with the potential options
│  │     ├─ python [2.7.15|3.6.7|3.7.1], which cannot be installed (as previously explained);
│  │     └─ python [3.7.0|3.7.1|...|3.7.9], which can be installed;
│  ├─ pybedtools [0.8.1|0.8.2|0.9.0] would require
│  │  ├─ python >=3.6,<3.7.0a0  with the potential options
│  │  │  ├─ python [2.7.15|3.6.7|3.7.1], which cannot be installed (as previously explained);
│  │  │  └─ python [3.6.0|3.6.1|...|3.6.9], which can be installed;
│  │  └─ python_abi 3.6.* *_cp36m, which can be installed;
│  ├─ pybedtools [0.8.1|0.8.2|0.9.0] would require
│  │  ├─ python >=3.7,<3.8.0a0  with the potential options
│  │  │  ├─ python [2.7.15|3.6.7|3.7.1], which cannot be installed (as previously explained);
│  │  │  └─ python [3.7.0|3.7.1|...|3.7.9], which can be installed;
│  │  └─ python_abi 3.7.* *_cp37m, which can be installed;
│  ├─ pybedtools [0.8.1|0.8.2|0.9.0|0.9.1] would require
│  │  ├─ python >=3.8,<3.9.0a0  with the potential options
│  │  │  ├─ python [3.8.0|3.8.1|...|3.8.8], which can be installed;
│  │  │  ├─ python [3.10.7|3.10.8|...|3.9.15] would require
│  │  │  │  └─ openssl >=1.1.1s,<1.1.2a , which can be installed;
│  │  │  ├─ python [3.10.7|3.10.8|...|3.9.16] would require
│  │  │  │  └─ openssl >=3.0.7,<4.0a0 , which can be installed;
│  │  │  ├─ python [3.8.16|3.9.16] would require
│  │  │  │  ├─ pypy3.8 7.3.11.*  with the potential options
│  │  │  │  │  ├─ pypy3.8 7.3.11 would require
│  │  │  │  │  │  └─ openssl >=3.0.7,<4.0a0 , which can be installed;
│  │  │  │  │  ├─ pypy3.8 7.3.11 would require
│  │  │  │  │  │  └─ openssl >=1.1.1s,<1.1.2a , which can be installed;
│  │  │  │  │  ├─ pypy3.8 7.3.11 would require
│  │  │  │  │  │  └─ openssl >=3.1.0,<4.0a0 , which can be installed;
│  │  │  │  │  ├─ pypy3.8 7.3.11 would require
│  │  │  │  │  │  └─ openssl >=3.1.2,<4.0a0 , which can be installed;
│  │  │  │  │  └─ pypy3.8 7.3.11 would require
│  │  │  │  │     └─ openssl >=3.1.3,<4.0a0 , which can be installed;
│  │  │  │  └─ pypy3.9 7.3.11.*  with the potential options
│  │  │  │     ├─ pypy3.9 7.3.11 would require
│  │  │  │     │  └─ openssl >=3.0.7,<4.0a0 , which can be installed;
│  │  │  │     ├─ pypy3.9 7.3.11 would require
│  │  │  │     │  └─ openssl >=3.1.0,<4.0a0 , which can be installed;
│  │  │  │     └─ pypy3.9 7.3.11 would require
│  │  │  │        └─ openssl >=1.1.1s,<1.1.2a , which can be installed;
│  │  │  ├─ python [3.10.12|3.8.17] would require
│  │  │  │  └─ openssl >=3.1.1,<4.0a0 , which can be installed;
│  │  │  └─ python [3.8.18|3.9.18] would require
│  │  │     ├─ openssl >=3.1.3,<4.0a0 , which can be installed;
│  │  │     └─ pypy3.9 7.3.13.*  with the potential options
│  │  │        ├─ pypy3.9 [7.3.12|7.3.13] would require
│  │  │        │  └─ openssl >=3.1.3,<4.0a0 , which can be installed;
│  │  │        └─ pypy3.9 7.3.13 would require
│  │  │           └─ openssl >=3.1.4,<4.0a0 , which can be installed;
│  │  └─ python_abi 3.8.* *_cp38, which can be installed;
│  ├─ pybedtools [0.8.2|0.9.0|0.9.1] would require
│  │  ├─ python >=3.9,<3.10.0a0  with the potential options
│  │  │  ├─ python [3.10.7|3.10.8|...|3.9.15], which can be installed (as previously explained);
│  │  │  ├─ python [3.10.7|3.10.8|...|3.9.16], which can be installed (as previously explained);
│  │  │  ├─ python [3.8.16|3.9.16], which can be installed (as previously explained);
│  │  │  ├─ python [3.8.18|3.9.18], which can be installed (as previously explained);
│  │  │  ├─ python [3.9.0|3.9.1|...|3.9.9], which can be installed;
│  │  │  └─ python [3.9.17|3.9.18] would require
│  │  │     ├─ openssl >=3.1.2,<4.0a0 , which can be installed;
│  │  │     └─ pypy3.9 7.3.12.*  with the potential options
│  │  │        ├─ pypy3.9 7.3.12 would require
│  │  │        │  └─ openssl >=3.1.2,<4.0a0 , which can be installed;
│  │  │        └─ pypy3.9 [7.3.12|7.3.13], which can be installed (as previously explained);
│  │  └─ python_abi 3.9.* *_cp39, which can be installed;
│  ├─ pybedtools 0.9.0 would require
│  │  ├─ pysam >=0.16  with the potential options
│  │  │  ├─ pysam [0.19.0|0.19.1] would require
│  │  │  │  ├─ libdeflate >=1.10,<1.11.0a0 , which conflicts with any installable versions previously reported;
│  │  │  │  ├─ python >=2.7,<2.8.0a0  with the potential options
│  │  │  │  │  ├─ python [2.7.12|2.7.13|...|2.7.18], which can be installed;
│  │  │  │  │  └─ python [2.7.15|3.6.7|3.7.1], which cannot be installed (as previously explained);
│  │  │  │  └─ python_abi 2.7.* *_cp27mu, which can be installed;
│  │  │  ├─ pysam [0.19.0|0.19.1] would require
│  │  │  │  ├─ libdeflate >=1.10,<1.11.0a0 , which conflicts with any installable versions previously reported;
│  │  │  │  ├─ python >=3.6,<3.7.0a0  with the potential options
│  │  │  │  │  ├─ python [2.7.15|3.6.7|3.7.1], which cannot be installed (as previously explained);
│  │  │  │  │  └─ python [3.6.0|3.6.1|...|3.6.9], which can be installed;
│  │  │  │  └─ python_abi 3.6.* *_cp36m, which can be installed;
│  │  │  ├─ pysam [0.19.0|0.19.1] would require
│  │  │  │  ├─ libdeflate >=1.10,<1.11.0a0 , which conflicts with any installable versions previously reported;
│  │  │  │  ├─ python >=3.7,<3.8.0a0  with the potential options
│  │  │  │  │  ├─ python [2.7.15|3.6.7|3.7.1], which cannot be installed (as previously explained);
│  │  │  │  │  └─ python [3.7.0|3.7.1|...|3.7.9], which can be installed;
│  │  │  │  └─ python_abi 3.7.* *_cp37m, which can be installed;
│  │  │  ├─ pysam [0.19.0|0.19.1] would require
│  │  │  │  ├─ libdeflate >=1.10,<1.11.0a0 , which conflicts with any installable versions previously reported;
│  │  │  │  ├─ python >=3.8,<3.9.0a0  with the potential options
│  │  │  │  │  ├─ python [3.8.0|3.8.1|...|3.8.8], which can be installed;
│  │  │  │  │  ├─ python [3.10.7|3.10.8|...|3.9.15], which can be installed (as previously explained);
│  │  │  │  │  ├─ python [3.10.7|3.10.8|...|3.9.16], which can be installed (as previously explained);
│  │  │  │  │  ├─ python [3.8.16|3.9.16], which can be installed (as previously explained);
│  │  │  │  │  ├─ python [3.10.12|3.8.17], which can be installed (as previously explained);
│  │  │  │  │  └─ python [3.8.18|3.9.18], which can be installed (as previously explained);
│  │  │  │  └─ python_abi 3.8.* *_cp38, which can be installed;
│  │  │  ├─ pysam [0.19.0|0.19.1] would require
│  │  │  │  ├─ libdeflate >=1.10,<1.11.0a0 , which conflicts with any installable versions previously reported;
│  │  │  │  ├─ python >=3.9,<3.10.0a0  with the potential options
│  │  │  │  │  ├─ python [3.10.7|3.10.8|...|3.9.15], which can be installed (as previously explained);
│  │  │  │  │  ├─ python [3.10.7|3.10.8|...|3.9.16], which can be installed (as previously explained);
│  │  │  │  │  ├─ python [3.8.16|3.9.16], which can be installed (as previously explained);
│  │  │  │  │  ├─ python [3.8.18|3.9.18], which can be installed (as previously explained);
│  │  │  │  │  ├─ python [3.9.0|3.9.1|...|3.9.9], which can be installed;
│  │  │  │  │  └─ python [3.9.17|3.9.18], which can be installed (as previously explained);
│  │  │  │  └─ python_abi 3.9.* *_cp39, which can be installed;
│  │  │  ├─ pysam 0.19.1 would require
│  │  │  │  ├─ libdeflate >=1.13,<1.14.0a0 , which conflicts with any installable versions previously reported;
│  │  │  │  └─ openssl >=1.1.1q,<1.1.2a , which can be installed;
│  │  │  ├─ pysam 0.19.1 would require
│  │  │  │  ├─ libdeflate >=1.10,<1.11.0a0 , which conflicts with any installable versions previously reported;
│  │  │  │  └─ openssl >=1.1.1q,<1.1.2a , which can be installed;
│  │  │  ├─ pysam [0.16.0|0.16.0.1] would require
│  │  │  │  └─ libdeflate >=1.6,<1.7.0a0 , which conflicts with any installable versions previously reported;
│  │  │  ├─ pysam [0.16.0.1|0.17.0] would require
│  │  │  │  └─ libdeflate >=1.7,<1.8.0a0 , which conflicts with any installable versions previously reported;
│  │  │  ├─ pysam [0.17.0|0.18.0] would require
│  │  │  │  └─ libdeflate >=1.9,<1.10.0a0 , which conflicts with any installable versions previously reported;
│  │  │  ├─ pysam 0.18.0 would require
│  │  │  │  ├─ python >=2.7,<2.8.0a0  with the potential options
│  │  │  │  │  ├─ python [2.7.12|2.7.13|...|2.7.18], which can be installed;
│  │  │  │  │  └─ python [2.7.15|3.6.7|3.7.1], which cannot be installed (as previously explained);
│  │  │  │  └─ python_abi 2.7.* *_cp27mu, which can be installed;
│  │  │  ├─ pysam 0.18.0 would require
│  │  │  │  ├─ python >=3.6,<3.7.0a0  with the potential options
│  │  │  │  │  ├─ python [2.7.15|3.6.7|3.7.1], which cannot be installed (as previously explained);
│  │  │  │  │  └─ python [3.6.0|3.6.1|...|3.6.9], which can be installed;
│  │  │  │  └─ python_abi 3.6.* *_cp36m, which can be installed;
│  │  │  ├─ pysam 0.18.0 would require
│  │  │  │  ├─ python >=3.7,<3.8.0a0  with the potential options
│  │  │  │  │  ├─ python [2.7.15|3.6.7|3.7.1], which cannot be installed (as previously explained);
│  │  │  │  │  └─ python [3.7.0|3.7.1|...|3.7.9], which can be installed;
│  │  │  │  └─ python_abi 3.7.* *_cp37m, which can be installed;
│  │  │  ├─ pysam 0.18.0 would require
│  │  │  │  ├─ python >=3.8,<3.9.0a0  with the potential options
│  │  │  │  │  ├─ python [3.8.0|3.8.1|...|3.8.8], which can be installed;
│  │  │  │  │  ├─ python [3.10.7|3.10.8|...|3.9.15], which can be installed (as previously explained);
│  │  │  │  │  ├─ python [3.10.7|3.10.8|...|3.9.16], which can be installed (as previously explained);
│  │  │  │  │  ├─ python [3.8.16|3.9.16], which can be installed (as previously explained);
│  │  │  │  │  ├─ python [3.10.12|3.8.17], which can be installed (as previously explained);
│  │  │  │  │  └─ python [3.8.18|3.9.18], which can be installed (as previously explained);
│  │  │  │  └─ python_abi 3.8.* *_cp38, which can be installed;
│  │  │  ├─ pysam 0.18.0 would require
│  │  │  │  ├─ python >=3.9,<3.10.0a0  with the potential options
│  │  │  │  │  ├─ python [3.10.7|3.10.8|...|3.9.15], which can be installed (as previously explained);
│  │  │  │  │  ├─ python [3.10.7|3.10.8|...|3.9.16], which can be installed (as previously explained);
│  │  │  │  │  ├─ python [3.8.16|3.9.16], which can be installed (as previously explained);
│  │  │  │  │  ├─ python [3.8.18|3.9.18], which can be installed (as previously explained);
│  │  │  │  │  ├─ python [3.9.0|3.9.1|...|3.9.9], which can be installed;
│  │  │  │  │  └─ python [3.9.17|3.9.18], which can be installed (as previously explained);
│  │  │  │  └─ python_abi 3.9.* *_cp39, which can be installed;
│  │  │  ├─ pysam 0.20.0 would require
│  │  │  │  └─ openssl >=1.1.1s,<1.1.2a , which can be installed;
│  │  │  ├─ pysam 0.21.0 would require
│  │  │  │  └─ openssl >=3.1.0,<4.0a0 , which can be installed;
│  │  │  ├─ pysam 0.21.0 would require
│  │  │  │  └─ openssl >=1.1.1t,<1.1.2a , which can be installed;
│  │  │  └─ pysam 0.22.0 would require
│  │  │     └─ openssl >=3.1.3,<4.0a0 , which can be installed;
│  │  └─ python >=3.10,<3.11.0a0  with the potential options
│  │     ├─ python [3.10.7|3.10.8|...|3.9.15], which can be installed (as previously explained);
│  │     ├─ python [3.10.7|3.10.8|...|3.9.16], which can be installed (as previously explained);
│  │     ├─ python [3.10.12|3.8.17], which can be installed (as previously explained);
│  │     ├─ python [3.10.0|3.10.10|...|3.10.9] conflicts with any installable versions previously reported;
│  │     ├─ python [3.10.0|3.10.1|3.10.2|3.10.4|3.10.5] would require
│  │     │  └─ python_abi 3.10.* *_cp310, which conflicts with any installable versions previously reported;
│  │     ├─ python 3.10.6 would require
│  │     │  └─ openssl >=1.1.1q,<1.1.2a , which can be installed;
│  │     ├─ python 3.10.6 would require
│  │     │  └─ openssl >=3.0.5,<4.0a0 , which can be installed;
│  │     ├─ python [3.10.10|3.10.11] would require
│  │     │  └─ openssl >=3.1.0,<4.0a0 , which can be installed;
│  │     └─ python 3.10.13 would require
│  │        └─ openssl >=3.1.4,<4.0a0 , which can be installed;
│  └─ pybedtools 0.9.1 would require
│     └─ pysam >=0.16  with the potential options
│        ├─ pysam [0.19.0|0.19.1], which cannot be installed (as previously explained);
│        ├─ pysam [0.19.0|0.19.1], which cannot be installed (as previously explained);
│        ├─ pysam [0.19.0|0.19.1], which cannot be installed (as previously explained);
│        ├─ pysam [0.19.0|0.19.1], which cannot be installed (as previously explained);
│        ├─ pysam [0.19.0|0.19.1], which cannot be installed (as previously explained);
│        ├─ pysam 0.19.1, which cannot be installed (as previously explained);
│        ├─ pysam 0.19.1, which cannot be installed (as previously explained);
│        ├─ pysam [0.16.0|0.16.0.1], which cannot be installed (as previously explained);
│        ├─ pysam [0.16.0.1|0.17.0], which cannot be installed (as previously explained);
│        ├─ pysam [0.17.0|0.18.0], which cannot be installed (as previously explained);
│        ├─ pysam 0.18.0, which can be installed (as previously explained);
│        ├─ pysam 0.18.0, which can be installed (as previously explained);
│        ├─ pysam 0.18.0, which can be installed (as previously explained);
│        ├─ pysam 0.18.0, which can be installed (as previously explained);
│        ├─ pysam 0.18.0, which can be installed (as previously explained);
│        ├─ pysam 0.20.0, which can be installed (as previously explained);
│        ├─ pysam 0.21.0, which can be installed (as previously explained);
│        ├─ pysam 0.21.0, which can be installed (as previously explained);
│        └─ pysam 0.22.0, which can be installed (as previously explained);
├─ pysam 0.19** , which cannot be installed (as previously explained);
├─ python-kaleido 0.2**  is not installable because it conflicts with any installable versions previously reported;
├─ python 3.10**  is installable with the potential options
│  ├─ python [3.10.7|3.10.8|...|3.9.15], which can be installed (as previously explained);
│  ├─ python [3.10.7|3.10.8|...|3.9.16], which can be installed (as previously explained);
│  ├─ python [3.10.12|3.8.17], which can be installed (as previously explained);
│  ├─ python [3.10.0|3.10.10|...|3.10.9] conflicts with any installable versions previously reported;
│  ├─ python [3.10.0|3.10.1|3.10.2|3.10.4|3.10.5], which cannot be installed (as previously explained);
│  ├─ python 3.10.6, which can be installed (as previously explained);
│  ├─ python 3.10.6, which can be installed (as previously explained);
│  ├─ python [3.10.10|3.10.11], which can be installed (as previously explained);
│  └─ python 3.10.13, which can be installed (as previously explained);
├─ requests 2.28**  is not installable because there are no viable options
│  ├─ requests [2.28.0|2.28.1|2.28.2] would require
│  │  └─ urllib3 >=1.21.1,<1.27 , which conflicts with any installable versions previously reported;
│  └─ requests [2.28.0|2.28.1] conflicts with any installable versions previously reported;
├─ samtools 1.16**  is not installable because there are no viable options
│  ├─ samtools 1.16.1 would require
│  │  └─ htslib >=1.17,<1.19.0a0  but there are no viable options
│  │     ├─ htslib 1.17 would require
│  │     │  └─ libcurl >=7.87.0,<8.0a0  but there are no viable options
│  │     │     ├─ libcurl [7.83.1|7.85.0|...|7.88.1] would require
│  │     │     │  └─ libnghttp2 [>=1.47.0,<2.0a0 |>=1.51.0,<2.0a0 |>=1.52.0,<2.0a0 ] but there are no viable options
│  │     │     │     ├─ libnghttp2 [1.47.0|1.51.0|1.52.0] would require
│  │     │     │     │  └─ c-ares >=1.18.1,<2.0a0 , which conflicts with any installable versions previously reported;
│  │     │     │     ├─ libnghttp2 1.55.1 would require
│  │     │     │     │  └─ c-ares >=1.20.1,<2.0a0 , which conflicts with any installable versions previously reported;
│  │     │     │     ├─ libnghttp2 1.58.0 would require
│  │     │     │     │  └─ c-ares >=1.21.0,<2.0a0 , which conflicts with any installable versions previously reported;
│  │     │     │     ├─ libnghttp2 1.58.0 would require
│  │     │     │     │  └─ c-ares >=1.23.0,<2.0a0 , which conflicts with any installable versions previously reported;
│  │     │     │     └─ libnghttp2 [1.52.0|1.57.0] conflicts with any installable versions previously reported;
│  │     │     └─ libcurl [7.87.0|7.88.1] conflicts with any installable versions previously reported;
│  │     └─ htslib [1.17|1.18] would require
│  │        └─ libdeflate >=1.18,<1.20.0a0 , which conflicts with any installable versions previously reported;
│  └─ samtools 1.16.1 would require
│     └─ htslib >=1.16,<1.19.0a0  but there are no viable options
│        ├─ htslib 1.16 would require
│        │  └─ libcurl >=7.83.1,<8.0a0  but there are no viable options
│        │     ├─ libcurl [7.83.1|7.85.0|...|7.88.1], which cannot be installed (as previously explained);
│        │     ├─ libcurl [7.84.0|7.85.0|7.86.0] conflicts with any installable versions previously reported;
│        │     └─ libcurl [7.87.0|7.88.1] conflicts with any installable versions previously reported;
│        ├─ htslib 1.17, which cannot be installed (as previously explained);
│        └─ htslib [1.17|1.18], which cannot be installed (as previously explained);
└─ ucsc-fatotwobit 377**  is not installable because there are no viable options
   ├─ ucsc-fatotwobit 377 would require
   │  └─ openssl >=1.0.2p,<1.0.3a , which conflicts with any installable versions previously reported;
   └─ ucsc-fatotwobit 377 would require
      └─ mysql-connector-c >=6.1.11,<6.1.12.0a0  but there are no viable options
         ├─ mysql-connector-c 6.1.11 conflicts with any installable versions previously reported;
         └─ mysql-connector-c 6.1.11 would require
            └─ openssl >=1.1.1,<1.1.2.0a0 , which does not exist (perhaps a missing channel).

It seems to me that the issue cannot be traced back to a single package. Moreover, unfortunately upgrading to Ubuntu 22 is currently not a valid option for me, since I did it before and had to downgrade back to Ubuntu 20 for several incompatibility issues with multiple other programs. Any ideas? Maybe a docker could work?

**Comment by BGSpiegl** [comment]
Please try the following:
- delete any conda cache files: `conda clean --all ; mamba clean --all`
- delete any previous environment: `mamba remove -n GCparagon --all`
- install environment with the YAML file from the new dev branch here:
  [https://github.com/BGSpiegl/GCparagon/blob/dev-Ubuntu20condaEnv/conda_env/GCparagon_py3.10_env.yml](url)
  `mamba env create -f GCparagon_py3.10_env.yml`
 - please report back what mamba outputs and if your issue was resolved or not!
 Right now I would prefer to solve this directly through loosened env specs.
 I might provide a docker/singularity image in the future though to get rid of these deploying annoyances once and for all.

**Comment by Rociocc93** [comment]
Hi! 

I tried again (with both solvers) and it didn't work. To try to understand the issue, I created a test conda environment using python=3.10, and iteratively added the indicated packages, starting with pybedtools (the first package libmamba complains about). My goal was to debug the issue step by step, but to my surprise, nothing went wrong. I don't really understand why the result was different, since in the new branch YAML you removed all version specifications except for the python ones, and so did I. 

In any case, it seems to be working just fine, so thank you again for your time! 

Rocío
